stable/hlf-peer/templates/pvc.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingClaim) }}
```

Comment:
Please follow our best practices for persistence. See https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md#persistence

Reply:
Updated, PTLA

Reply:
@unguiculus, updated, PTLA